File: src/main/java/org/gdg/withgo/service/Postgresql.java
package org.gdg.withgo.service;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class Postgresql {

    public static Connection create() throws SQLException {
        String dbHost = System.getenv("DB_HOST");
        String dbName = System.getenv("DB_NAME");
        String dbPort = System.getenv("DB_PORT");
        String dbUser = System.getenv("DB_USER");
        String dbPassword = System.getenv("DB_PASSWORD");
        String url = String.format("jdbc:postgresql://%s:%s/%s", dbHost,dbPort,dbName);
        return DriverManager.getConnection(url,dbUser, dbPassword);
    }
}

File: src/main/java/org/gdg/withgo/service/Postgresql.kt
package org.gdg.withgo.service

import org.jooq.impl.DSL
import java.sql.Connection
import java.sql.DriverManager
import java.sql.SQLException

object Postgresql {

    @Throws(SQLException::class)
    fun create(): Connection {
        val dbHost = System.getenv("DB_HOST")
        val dbName = System.getenv("DB_NAME")
        val dbPort = System.getenv("DB_PORT")
        val dbUser = System.getenv("DB_USER")
        val dbPassword = System.getenv("DB_PASSWORD")
        val url = "jdbc:postgresql://$dbHost:$dbPort/$dbName"
        return DriverManager.getConnection(url, dbUser, dbPassword)
    }

    fun dsl() = DSL.using(create())
}
